src/assets/js/shared/components/MusicTile.css

```diff
@@ -0,0 +1,17 @@
+.musicTile--container {
```

Comment:
So typically you can just set the root container's css to the component/file name - `.music-tile`. and then build up the classnames from that